pkg/services/sqlstore/stats.go

```diff
@@ -183,3 +184,18 @@ func GetSystemUserCountStats(ctx context.Context, query *models.GetSystemUserCou
 		return err
 	})
 }
+
+func GetActiveUserCountStats(query *models.GetActiveUserCountStatsQuery) error {
+	activeEndDate := time.Now().Add(-activeUserTimeLimit)
+
+	var rawSql = `SELECT COUNT(*) AS Count FROM ` + dialect.Quote("user") + ` where last_seen_at > ?`
```

Comment:
Consistent SQL formatting:
```suggestion
	var rawSql = `SELECT COUNT(*) AS Count FROM ` + dialect.Quote("user") + ` WHERE last_seen_at > ?`
```